src/collections/news/2020-05-28-layer5-and-hasicorp-launch-service-mesh-partnership/index.mdx

```diff
@@ -15,7 +15,7 @@ import layer5_hashicorp_partnership from "./Layer5-HashiCorp-Service-Mesh-Partne
 import layer5_image_hub from "./layer5-image-hub-on-hashicorp-consul.png";
 
 <NewsWrapper>
-
+<link rel="canonical" href="/company/news/layer5-and-hashicorp-launch-service-mesh-partnership" />
```

Comment:
Why this change?

Reply:
Ok, I think Lee added this....merging 👍🏼

Reply:
Oops, yes, I was checking out @chinmaym07's update and fixed another item while in there, not realizing I was pushing to Chinmay's branch.